src/subdomains/generic/user/models/auth/auth.service.ts

```diff
@@ -54,7 +54,9 @@ export class AuthService {
 
   // --- AUTH METHODS --- //
   async signUp(dto: CreateUserDto, userIp: string): Promise<{ accessToken: string }> {
```

Comment:
We need to verify the provided blockchain fits the address. Maybe add as parameter to verifySignature?